packages/server/src/tests/banners/bannerDeploySchedule.ts

```diff
@@ -0,0 +1,63 @@
+import { set } from 'date-fns';
```

Comment:
I do wonder if `set` could be renamed to something more meaningful, not sure what

Reply:
pushing up a tiny refactor to make use of these...